test/screenshot/README.md

```diff
@@ -1,22 +1,144 @@
 # Screenshot Tests
 
-## Local dev server
+Prevent visual regressions by running screenshot tests on every PR.
 
-Run:
+## Quick start
+
+Add the following to your `~/.bash_profile` (Mac) or `~/.bashrc` (Linux):
```

Comment:
Both of these files work on both OSes, so I'm not sure these parentheticals are warranted... bash_profile vs. bashrc is a login-shell vs. interactive-non-login-shell thing. I think most terminal apps tend to execute login shells?

Reply:
Done